services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationController.kt

```diff
@@ -280,4 +314,11 @@ class ApplicationController(
         logger.warn(ex) { ex.message }
         return HankeError.HAI2006
     }
+
+    @ExceptionHandler(HankeNotFoundException::class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    fun missingApplication(ex: HankeNotFoundException): HankeError {
+        logger.warn(ex) { ex.message }
+        return HankeError.HAI1001
+    }
```

Comment:
The same handler is already defined in `ControllerExceptionHandler`, which is used if there's no controller-specific handler for an exception.

Reply:
Removed.